helm/values.yaml

```diff
@@ -45,6 +45,9 @@ notary:
   rootPubKey: |
     -----BEGIN PUBLIC KEY-----
     -----END PUBLIC KEY-----
+  # whether the notary server will be the ACR
```

Comment:
This seems insufficiently explicit. I would recommend to write it out:
`# if you use Azure Container Registry (ACR) for your notary changes ... . For compatibility with ACR set isAcr to `true`